EIPS/eip-888.md

```diff
@@ -0,0 +1,178 @@
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+Every ERC-888 compliant contract MUST implement the ERC888 and ERC165 interfaces:
+
+```
+// SPDX-License-Identifier: CC0
+
+pragma solidity ^0.8.0;
+
+/// @title ERC-888 EXP Token Standard
+/// @dev See https://eips.ethereum.org/EIPS/EIP-888
+///  Note: the ERC-165 identifier for this interface is ###ERC888###.
+///  Must initialize contracts with an `_operator` address that is not `address(0)`.
+///  Must initialize contracts assigning participation approval for both `_operator` and `address(0)`.
+interface IERC888 /* is ERC165 */ {
+
+    /// Emits when operator changes.
+    /// @dev MUST emit whenever operator changes.
+    event Appointment(address indexed _operator);
+
+    /// Emits when an address opts to participate.
+    /// @dev MUST emit whenever an address begins or ends participation.
+    ///  Transfers SHOULD NOT reset participation.
+    event Approval(address indexed _participant, bool _participation);
+
+    /// @notice Emits when operator transfers EXP to participating address. 
+    /// @dev MUST emit when EXP is created (`from` == 0), 
+    ///  destroyed (`to` == 0), or reallocated to another address.
+    ///  Exception: during contract creation, any amount of EXP
+    ///  MAY be created and assigned without emitting Transfer. 
+    event Transfer(address indexed _from, address indexed _to, uint256 _amount);
+
+    /// @notice Returns total EXP allocated to a participant.
+    /// @dev Should first assert queried address is participating.
+    /// @param _participant An address for whom to query EXP total
+    /// @return uint256 The number of EXP allocated to `_participant`, possibly zero.
+    function balanceOf(address _participant) external view returns (uint256);
+
+    /// @notice Transfers EXP from zero address to a participant.
+    /// @dev MUST throw unless msg.sender is operator.
+    ///  MUST throw unless _to address is participating.
+    function transfer(address _to, uint256 _amount) external;
+
+    /// @notice Transfer EXP from one address to another.
+    /// @dev MUST throw unless msg.sender is operator.
+    ///  MUST throw unless _to address is participating.
+    ///  MAY throw if _from address is NOT participating.
+    function transferFrom(address _from, address _to, uint256 _amount) external;
+
+    /// @notice Activate or deactivate participation.
+    /// @dev MUST throw unless msg.sender is _participant.
+    ///  MUST throw if _participant is _operator or zero address.
+    /// @param _participant Address opting in or out of participation.
+    /// @param _participation Participation status of _participant.
+    function approve(address _participant, bool _participation) external;
```

Comment:
being that `approve` has a bit of different meaning in ERC-20 and ERC-721, I'd recommend a different nomenclature

Reply:
That's a fair point. I'm trying to minimize new terminology, but if the actual functions are distinct enough I'm happy to change.

Going with the general thrust of terminology in the EIP, perhaps `participate` is a more natural name for the function, and `Participate` for the event. Other options might be `consent` or `join`. What do folks think? 

I'll update to `participate` if there's no more feedback in a day or two.

Reply:
Updated with this nomenclature in the latest commit.

Reply:
Further discussion on this topic on the [Ethereum Magicians thread](https://ethereum-magicians.org/t/eip-4974-fungible-non-tradable-tokens-or-exp/8805/10)